buildpack/databroker/config_generator/templates/jmx/kafka_connect.yaml

```diff
@@ -0,0 +1,208 @@
+      tags:
+          connector: $1
+          task: $2
+      bean_regex: 'kafka\.connect:type=task-error-metrics,connector=(.*?),task=(.*?)(?:,|$)'
+  - include:
+      attribute:
+        LastEvent:
+          alias: debezium.postgres.snapshot.last-event
+        MilliSecondsSinceLastEvent:
+          alias: debezium.postgres.snapshot.milliseconds-since-last-event
+        MonitoredTables:
+          alias: debezium.postgres.snapshot.monitored-tables
+        NumberOfEventsFiltered:
+          alias: debezium.postgres.snapshot.number-of-events-filtered
+        QueueRemainingCapacity:
+          alias: debezium.postgres.snapshot.queue-remaining-capacity
+        QueueTotalCapacity:
+          alias: debezium.postgres.snapshot.queue-total-capacity
+        RemainingTableCount:
+          alias: debezium.postgres.snapshot.remaining-table-count
+        RowsScanned:
+          alias: debezium.postgres.snapshot.rows-scanned
+        SnapshotAborted:
+          alias: debezium.postgres.snapshot.snapshot-aborted
+        SnapshotCompleted:
+          alias: debezium.postgres.snapshot.snapshot-completed
+        SnapshotDurationInSeconds:
+          alias: debezium.postgres.snapshot.snapshot-duration-in-seconds
+        SnapshotRunning:
+          alias: debezium.postgres.snapshot.snapshot-running
+        TotalNumberOfEventsSeen:
+          alias: debezium.postgres.snapshot.total-number-of-events-seen
+        TotalTableCount:
+          alias: debezium.postgres.snapshot.total-table-count
+        bean_regex: 'debezium\.postgres:type=connector-metrics,context=snapshot,server=.*'
+  - include:
+      attribute:
+        Connected:
+          alias: debezium.postgres.streaming.connected
+        LastEvent:
+          alias: debezium.postgres.streaming.last-event
+        LastTransactionId:
+          alias: debezium.postgres.streaming.last-transaction-id
+        MilliSecondsBehindSource:
+          alias: debezium.postgres.streaming.milliseconds-behind-source
+        MilliSecondsSinceLastEvent:
+          alias: debezium.postgres.streaming.milliseconds-since-last-event
+        MonitoredTables:
+          alias: debezium.postgres.streaming.monitored-tables
+        NumberOfCommittedTransactions:
+          alias: debezium.postgres.streaming.number-of-committed-transactions
+        NumberOfEventsFiltered:
+          alias: debezium.postgres.streaming.number-of-events-filtered
+        QueueRemainingCapacity:
+          alias: debezium.postgres.streaming.queue-remaining-capacity
+        QueueTotalCapacity:
+          alias: debezium.postgres.streaming.queue-total-capacity
+        SourceEventPosition:
+          alias: debezium.postgres.streaming.source-event-position
+        TotalNumberOfEventsSeen:
+          alias: debezium.postgres.streaming.total-number-of-events-seen
+      bean_regex: 'debezium\.postgres:type=connector-metrics,context=streaming,server=.*'
+  host: localhost
+  port: 11003
+  java_bin_path: /home/vcap/app/.local/bin/java
```

Comment:
This is a hardcoded path, which will cause problems in the future. Please consider making this a template value.

Reply:
this has been changed to a template value